snow/engine/snowman/transitive.go

```diff
@@ -106,7 +102,7 @@ type Transitive struct {
 	errs wrappers.Errs
 }
 
-func newTransitive(config Config) (*Transitive, error) {
+func NewTransitive(config Config) (*Transitive, error) {
```

Comment:
I feel like this should just be "New" we only called it "newTransitive" to avoid shadowing "new"